WeakAuras/Prototypes.lua

```diff
@@ -1391,11 +1391,11 @@ Private.load_prototype = {
       name = "petbattle",
       display = L["In Pet Battle"],
       type = "tristate",
-      init = WeakAuras.IsMistsOrRetail() and "arg" or nil,
+      init = WeakAuras.IsWrathOrMistsOrRetail() and "arg" or nil,
```

Comment:
Do Pet Battles exist in Wrath?

Reply:
Yes, in Titan Reforged they’re shut be implemented, even though they weren’t part of the original Wrath expansion. The API works, i haven't seen a Pet Battle yet tho.

Reply:
No it does not. Because Wrath CN forked from 3.4.3, later on adapted to 3.4.4 (Cata but with wrath contents) and 3.4.5 (MoP with only wrath contents), you can find those API but the content isn't available.

Reply:
Purged, Thank you for the information!